vignettes/corridor-delineation.Rmd

```diff
@@ -11,54 +11,46 @@ vignette: >
 knitr::opts_chunk$set(
   collapse = TRUE,
   comment = "#>",
-  warning = FALSE,
-  eval = FALSE  #TODO remove this to enable code chunks after #87 is merged
+  warning = FALSE
 )
 ```
 
 ```{r setup, warning=FALSE, message=FALSE}
 library(CRiSp)
+library(sf)
```

Comment:
I think it is okay to attach sf here as it is unlikely the user does not have it when working with data to/from CRiSp.